.buildkite/pipelines/security_solution/api_integration.yml

```diff
@@ -76,6 +76,40 @@ steps:
         - exit_status: '1'
           limit: 2
 
+  - label: Running rule_creation:essentials:qa:serverless
+    command: .buildkite/scripts/pipelines/security_solution_quality_gate/api-integration-tests.sh rule_creation:essentials:qa:serverless
+    key: rule_creation:essentials:qa:serverless
+    agents:
+      queue: n2-4-spot
+    timeout_in_minutes: 120
+    retry:
+      automatic:
+        - exit_status: '1'
```

Comment:
Don't want to block, makes more sense as a follow up for the whole file:

It looks like we don't have any retries for spot instance preemptions.  The exit status is `-1` in that case.  Depending on stability requirements we can swap out spot instances or add retries.

Reply:
@MadameSheema any feedback on what the suggested value is?